src/sets.jl

```diff
@@ -411,6 +456,8 @@ struct PositiveSemidefiniteConeSquare <: AbstractSymmetricMatrixSetSquare
     side_dimension::Int
 end
 
+dual_set(s::PositiveSemidefiniteConeSquare) = error("Dual of $s is not defined, $s can be a non-covex set.")
```

Comment:
How can `PositiveSemidefiniteConeSquare` be a non-convex set?

Reply:
I misunderstood it, I thought it could be non-symmetric but in fact, in MOI documentation it explains that the matrix is considered to be symmetric and positive semidefinite.